commit 454e2dbe1b4f
Author: Mike Shal <mshal@mozilla.com>
Date:   Tue Dec 20 16:37:18 2016 -0500

    Bug 1320991 - Support --with-system-{nss,nspr} in modules/libmar; r?glandium
    
    The problem with the modules/libmar/tests/moz.build file when building
    --with-system-nspr and --with-system-nss is that the nss libraries don't
    exist in the tree, so they fail when trying to copy into the test
    directory.
    
    However, it turns out that the libraries copied into the test directory
    aren't even used when building with an in-tree copy, because the
    xpcshell launcher sets LD_LIBRARY_PATH to point to dist/bin. Since we
    use the dist/bin copies anyway for an in-tree build, we can stop copying
    them into the test directory and simultaneously fix the --with-system
    build.
    
    The DEFINES can also go away since this directory doesn't actually build
    anything.
    
    MozReview-Commit-ID: Bk2f28wc9ZJ
---
 modules/libmar/tests/moz.build | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git modules/libmar/tests/moz.build modules/libmar/tests/moz.build
index ee72571b0c6b..9642553e8aa2 100644
--- mozilla/modules/libmar/tests/moz.build
+++ mozilla/modules/libmar/tests/moz.build
@@ -6,24 +6,7 @@
 
 XPCSHELL_TESTS_MANIFESTS += ['unit/xpcshell.ini']
 
-DEFINES['BIN_SUFFIX'] = CONFIG['BIN_SUFFIX']
-
 if CONFIG['OS_TARGET'] != 'Android':
-    nss_files = ['nss3']
-    if not CONFIG['MOZ_FOLD_LIBS']:
-        nss_files.extend([
-            'nssutil3',
-            'plc4',
-            'nspr4',
-            'plds4',
-        ])
-    TEST_HARNESS_FILES.xpcshell.modules.libmar.tests.unit += sorted(
-        ['!/dist/bin/%s%s%s' % (
-            CONFIG['DLL_PREFIX'],
-            name,
-            CONFIG['DLL_SUFFIX'],
-        ) for name in nss_files]
-    )
     TEST_HARNESS_FILES.xpcshell.modules.libmar.tests.unit += [
         '!../tool/signmar%s' % CONFIG['BIN_SUFFIX'],
     ]
